### PR TITLE
[ch160840] filter count styles

### PR DESCRIPTION
**Pull request by mmmmmmaya**
### Summary
- **What:** Style the filter panel toggle button
- **Ticket:** [[ch160840]](https://app.clubhouse.io/genepi/story/160840)
- **Env:** https://filterbadge-frontend.dev.genepi.czi.technology

### Demos
after

### Notes
With the tooltip on top of the icon, it looks a bit weird and clashes with the nav bar. I made a choice to put the tooltip below, but open to changing it.
Screen Shot 2021-09-17 at 9 39 38 AM Screen Shot 2021-09-17 at 9 46 58 AM

### Checklist
- [x] I merged latest `feat/filtering`
- [x] I manually verified the change
- [x] I added labels to my PR
- [ ] I tested in multiple browsers
- [ ] I added relevant unit tests
- [ ] I have notified others of changes they need to make locally (migrations, jobs, package updates, etc)

### CODE REVIEW

**Review by vincent-czi — APPROVED** [review]

File: src/frontend/src/views/Data/components/FilterPanelToggle/style.ts [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
 import styled from "@emotion/styled";
-import { getColors, getSpacings } from "czifui";
-import IconFilters from "src/common/icons/IconFilters.svg";
+import { fontHeaderXxxs, getColors, getCorners, getSpacings } from "czifui";
 
 export const StyledDiv = styled.div`
   display: flex;
   align-items: center;
-  justify-content: center;
+  align-content: center;
```
Comment (phoenixAja) [inline src/frontend/src/views/Data/components/FilterPanelToggle/style.ts]: do we need both align-items and align-content?

File: src/frontend/src/views/Data/components/FilterPanelToggle/style.ts [hunk 2]
```diff
@@ -30,14 +31,32 @@ export const StyledDiv = styled.div`
   }
 `;
 
-export const StyledIconFilters = styled(IconFilters)`
+export const StyledBadge = styled.div`
+  ${fontHeaderXxxs}
+
```
Comment (phoenixAja) [inline src/frontend/src/views/Data/components/FilterPanelToggle/style.ts:35]: nit: idt we need this new line here

**Review by phoenixAja — APPROVED** [review]
lg!

File: src/frontend/src/views/Data/components/FilterPanelToggle/style.ts (hunk 1 above)
Comment (mmmmmmaya) [reply src/frontend/src/views/Data/components/FilterPanelToggle/style.ts:8]: yes, one centers vertically and one centers horizontally

Comment (mmmmmmaya) [reply src/frontend/src/views/Data/components/FilterPanelToggle/style.ts]: oops, i must have accidentally added the extra one here when I was trying ot update the badge

**Review by Janeece — APPROVED** [review]
@mmmmmmaya Looks beautiful, no changes! thank you 😊

**Pull request merged**